File: 8kyu-grasshopper-Messi-goals-function.jl
# Grasshopper - Messi goals function
# Language: Julia
# Version 1.0
# Difficulty: 8 kyu

function goals(laLigaGoals, copaDelReyGoals, championsLeagueGoals) 
  laLigaGoals + copaDelReyGoals + championsLeagueGoals
end


# Tests
using FactCheck

facts("goal") do
  context("Sample Tests") do
    @fact goals(0, 0, 0) --> 0
    @fact goals(43, 10, 5) --> 58
  end
end

File: 8kyu-grasshopper-Messi-goals-function.py
# Grasshopper - Messi goals function
# Language: Python
# Version 3.10
# Difficulty: 8 kyu

def goals(laLiga, copaDelRey, championsLeague):
    return laLiga + copaDelRey + championsLeague


# Tests
import codewars_test as test
from solution import goals

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(goals(0, 0, 0), 0)
        test.assert_equals(goals(5, 10, 2), 17)